site/en/posts/desktop-avd-in-android-studio.md

```diff
@@ -0,0 +1,89 @@
+
+Large screen Android devices are becoming an increasingly popular way for users to run Android apps, with a 50% YoY growth[^1] of users engaging with Android apps on Chrome OS last year. However, the Android app experience on Chromebooks is not always completely satisfying for users, as most Android apps and games are not optimized for large screens. Apps run into a variety of compatibility issues which directly impact user experience and satisfaction.
+
+As more developers optimize their apps for Chromebooks, they are looking to test and validate their app experience on Chromebooks. Today, that journey for developers includes purchasing a physical Chromebook device, setting it up in their development environment, and finally deploying the app to the Chromebook for testing. That’s a lot of friction. We agree. It shouldn’t be that hard. With the launch of Desktop AVD, developers can now test their apps in a desktop environment without all that extra work.
+
+## Running apps on an emulator
+
+Android apps can run on a variety of form factors, and it’s impractical for developers to physically own each device. That’s why, for years, Android Studio has offered Android developers the ability to install and test apps on emulators across different form factors, including phones, tablets, Wear OS, Android TV, and Android Auto devices. This has enabled developers to simulate both hardware and software characteristics of these Android devices and environments. In addition to these form factors, we are expanding the emulator support to include desktop devices that can run Android apps, such as Chromebooks.
+
+With the Desktop AVD in Android Studio, Android developers can get a taste of how Android apps function on a desktop, including some unique characteristics of desktops such as freeform window resizing and drag and drop. This enables developers to validate that their app responds accurately to app resize events, renders fine in a landscape mode, and keyboard and mouse input works as expected across the app. We have included a handful of things below you should test with your app on the Desktop AVD.
+
+By validating the app experience on the Desktop AVD, developers can feel confident in providing a superior user experience to users on Chromebooks.
+
+## Desktop emulator
+
+In the Desktop AVD, apps launch in freeform windowing mode. In this mode, all apps show a caption bar allowing actions like maximize, minimize and close. Desktop AVD also introduces a taskbar on the bottom of the screen that you can use to switch between running apps by clicking their app icon. Quick settings and notification panels can be accessed from the system tray in the bottom right, instead of the traditional swipe down from top action on other touch-first devices.
+
+![Desktop Android Virtual Device showing three apps running simulatenously in their respective windows.](ix://posts/desktop-avd-in-android-studio/freeform_windows.png)
+
+## Try it out yourself
```

Comment:
As a heads up just cuz I got a notification and checked out the post. When I read this headline I immediately thought it was going to tell me how I could try out the emulator. Maybe wording it to something like "See the difference yourself", especially cuz we say try out the emulator later.

Just a thought!

Reply:
Thanks for the feedback! The team decided they wanted to keep it as-is for launch.